### DISCUSSION TITLE
std::string and fstream error

### DISCUSSION THREAD

**Discussion by sheershak-agarwal** [nasa/fprime#786] [Q&A]
Hi,

I am trying use std::string and fstream libraries in f-prime but while compiling it throws an error. Can't I use external libraries with fprime? Or do I have change some settings to enable the use of external libraries?

**Comment by Joshua-Anderson** [comment]
What target are you compiling for? While F' core avoids usage of the stdlib for compatibility reasons, there's no reason why users can't use it. Since they're part of the compiler you shouldn't need any specific configuration to use them. What errors are you seeing?
